src/components/Menu.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './Menu.module.css';
+
+const Menu = props => (
+  <div>
+    <select className={styles.chooseRecipe}id="recipe-select" onChange={props.onChange}>
```

Comment:
Have you got a ticket to drive the values in this select with data rather than hard coded?